### PR TITLE
KG assay data and updated templates

### PR DESCRIPTION
**Pull request by MicPie**

### CODE REVIEW

File: data/tabular/carcinogens/transform.py [hunk 1]
```diff
@@ -135,6 +135,78 @@ def get_and_transform_data():
 title = {admetSAR: A Comprehensive Source and Free Tool for Assessment of Chemical ADMET Properties},
 journal = {Journal of Chemical Information and Modeling}""",
         ],
+        "templates": [
+            "The molecule with the {SMILES__description} {#representation of |!}{SMILES#} {#shows|exhibits|displays!} {carcinogen#no &NULL}{carcinogen__names__adjective} properties.",  # noqa: E501
```
Comment (AdrianM0) [inline data/tabular/carcinogens/transform.py:139]:
```suggestion
            "The molecule with the {SMILES__description} {#representation of |!}{SMILES#} {#shows|exhibits|displays!} {carcinogen#no &NULL}{carcinogen__names__adjective} effects.",  # noqa: E501
```

File: data/tabular/carcinogens/transform.py [hunk 2]
```diff
@@ -135,6 +135,78 @@ def get_and_transform_data():
...
+            "Based on the {SMILES__description} {#representation |!}{SMILES#}, the molecule has {carcinogen#no &NULL}{carcinogen__names__adjective} {#properties|characteristics|features!}.",  # noqa: E501
```
Comment (AdrianM0) [inline data/tabular/carcinogens/transform.py:140]:
```suggestion
            "Based on the {SMILES__description} {#representation |!}{SMILES#}, the molecule has {carcinogen#no &NULL}{carcinogen__names__adjective} {#effects|implications|impact!}.",  # noqa: E501
```

File: data/tabular/clintox/transform.py [hunk 3]
```diff
@@ -100,6 +100,77 @@ def get_and_transform_data():
 title = {A Data-Driven Approach to Predicting Successes and Failures of Clinical Trials},
 journal = {Cell Chemical Biology}}""",
         ],
+        "templates": [
+            "The molecule with the {SMILES__description} {#representation of |!}{SMILES#} {#shows|exhibits|displays!} {clinical_toxicity#no &NULL}{clinical_toxicity__names__noun}.",  # noqa: E501
+            "Based on the {SMILES__description} {#representation |!}{SMILES#}, the molecule has {clinical_toxicity#no &NULL}{clinical_toxicity__names__adjective} {#properties|characteristics|features!}.",  # noqa: E501
```
Comment (AdrianM0) [inline data/tabular/clintox/transform.py:105]:
```suggestion
            "Based on the {SMILES__description} {#representation |!}{SMILES#}, the molecule has {clinical_toxicity#no &NULL}{clinical_toxicity__names__adjective} {#properties|characteristics|traits!}.",  # noqa: E501
```

File: data/tabular/carcinogens/transform.py (hunk 1 above)
Comment (MicPie) [reply data/tabular/carcinogens/transform.py:139]: Thank you, I added both. 👍

File: data/tabular/carcinogens/transform.py (hunk 2 above)
Comment (MicPie) [reply data/tabular/carcinogens/transform.py:140]: Thank you, I added a mix. 👍

File: data/tabular/clintox/transform.py (hunk 3 above)
Comment (MicPie) [reply data/tabular/clintox/transform.py:105]: Thank you, I added a mix. 👍

**Review by AdrianM0 — APPROVED** [review]
lgtm

**Comment by MicPie** [comment]
For dimensionless units = `"units": None,` we need to adapt the code or add a placeholder.

Tabular datasets w/o units:
lipophilicity

File: data/tabular/bioavailability_ma_et_al/transform.py [hunk 4]
```diff
@@ -119,22 +119,76 @@ def get_and_transform_data():
 journal = {Journal of Pharmaceutical and Biomedical Analysis}""",
         ],
         "templates": [
-            "The molecule with the {SMILES__description} representation {SMILES#} has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
-            "Based on the {SMILES__description} representation {SMILES#}, the molecule has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
-            "The {SMILES__description} {SMILES#} represents a molecule that has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
+            "The molecule with the {SMILES__description} {#representation of |!}{SMILES#} has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
+            "Based on the {SMILES__description} {#representation of |!}{SMILES#}, the molecule has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
+            "The {SMILES__description} {SMILES#} {#represents|is from!} a molecule that has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
```
Comment (AdrianM0) [inline data/tabular/bioavailability_ma_et_al/transform.py:124]: ```SMILES is from``` a molecule doesn't sound too good to me. I propose:

```suggestion
            "The {SMILES__description} {SMILES#} {#represents|is!} a molecule that has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
```

File: data/tabular/blood_brain_barrier_martins_et_al/meta.yaml [hunk 5]
```diff
@@ -72,84 +72,84 @@ bibtex:
       title = {MoleculeNet: a benchmark for molecular machine learning},
       journal = {Chemical Science}
 templates:
-    - The molecule with the {SMILES__description} of {SMILES#} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
-    - Based on the {SMILES__description} {SMILES#}, the molecule is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
-    - The {SMILES__description} {SMILES#} represents a molecule that is {penetrate_BBB#not &NULL}identified as {penetrate_BBB__names__adjective}.
+    - The molecule with the {SMILES__description} {SMILES#} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
+    - Based on the {SMILES__description} {#representation |!}{SMILES#}, the molecule is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
+    - The {SMILES__description} {SMILES#} {#represents|is from!} a molecule that is {penetrate_BBB#not &NULL}identified as {penetrate_BBB__names__adjective}.
```
Comment (AdrianM0) [inline data/tabular/blood_brain_barrier_martins_et_al/meta.yaml:77]: same as above

**Review by AdrianM0 — COMMENTED** [review]
looks good, there is one small comment related to the "SMILES is from a molecule" that appears in many files. I searched it on google scholar and there is no such formulation anywhere.

**Review by AdrianM0 — APPROVED** [review]

**Pull request merged**